clients/index.js

```diff
@@ -417,3 +418,17 @@ ApplicationClients.prototype.updateKValues = function updateKValues() {
         self.serviceProxy.updateServiceChannels();
     }
 };
+
+ApplicationClients.prototype.updateKillSwitches = function updateKillSwitches() {
+    var self = this;
+    self.serviceProxy.unblockAll();
```

Comment:
I also don't like this approach of: "unblock all, then block these".

Please instead: "block these, then unblock any that I didn't just block".